docs/create_executable.md

```diff
@@ -33,4 +33,8 @@ ToDo
 
 ## Mac
 
```

Comment:
Add the necessary steps before the command used to generate the executable. Like we did for Windows

> Follow these steps to generate the executable for Windows.
> 
> 1. Run these commands
> 
> ```
> cd path/to/dot
> conda activate dot
> ```
> 
> 2. Get the path of the `site-packages` by running this command
> 
> ```
> python -c "import site; print(''.join(site.getsitepackages()))"
> ```
> 
> 3. Replace `path/to/site-packages` with the path of the `site-packages` and run this command